objc/core/A0LockReact.m

```diff
@@ -44,6 +49,19 @@ - (void)configureLockWithClientId:(NSString *)clientId domain:(NSString *)domain
     _lock = [A0Lock newLockWithClientId:clientId domain:domain];
 }
 
+- (void)hideWithCallback:(A0LockCallback)callback {
+    if (self.shown) {
```

Comment:
I see, I still don't like it and probably is something the app might take care of (trying not to dismiss something is not there) but besides my opinion there is other argument against it. So let's keep it as it is, just check the other comment and I'll merge it